integration/run_test.go

```diff
@@ -140,6 +154,9 @@ func TestRun(t *testing.T) {
 			args := []string{"run"}
 			args = append(args, testCase.extraArgs...)
 			cmd := exec.Command("skaffold", args...)
+			for k, v := range testCase.env {
+				cmd.Env = append(cmd.Env, fmt.Sprintf("%s=%s", k, v))
```

Comment:
if nil this will use the process's env

if not, it will only use what you pass in. maybe we can just use the Setenv test function we have